generators/server/files.js

```diff
@@ -35,7 +35,6 @@ function writeFiles() {
             this.copy('gitignore', '.gitignore');
             this.copy('gitattributes', '.gitattributes');
             this.copy('editorconfig', '.editorconfig');
-            this.template('_travis.yml', '.travis.yml', this, {});
             this.template('_Jenkinsfile', 'Jenkinsfile', this, {});
```

Comment:
Shouldn't _Jenkinsfile be removed from main generator and from here then also?
If yes then also remove from expected test files.